crates/turbopack/src/module_options/module_rule.rs

```diff
@@ -73,54 +67,53 @@ impl ModuleRuleCondition {
     }
 }
 
-#[turbo_tasks::value_impl]
 impl ModuleRuleConditionVc {
-    #[turbo_tasks::function]
-    pub async fn matches(self, path: FileSystemPathVc) -> Result<BoolVc> {
+    pub async fn matches(self, path: FileSystemPathVc) -> Result<bool> {
```

Comment:
This actually seems expensive to recompute continually.

Reply:
It will only get recomputed if the function that calls this gets recomputed. At the moment, this is cheaper than the turbo_tasks overhead.

We might want to turn the top-level `ModuleRule::matches` back into a `turbo_tasks::function` at some point, but for now this is faster.